pkgs/games/domination/default.nix

```diff
@@ -52,37 +53,37 @@ in stdenv.mkDerivation {
     runHook postBuild
   '';
 
-  desktopItems = [
-    desktopItem
-    editorDesktopItem
-  ];
-
   installPhase = ''
     runHook preInstall
+
+    pushd build/game
```

Comment:
IMHO, stateful commands like pushd/popd makes the installPhase more difficult to read.

Reply:
If you say that I should, I'll revert everything except the `meta` and `postPatch` changes

Reply:
It's a weak recommendation from my side :-) I'm not maintaining this code, so my opinion doesn't matter as much as the ones who do (maintain it).

Reply:
Please do, I agree with @bjornfor